bpm/bonita-api/bonita-common-api-impl/src/main/java/org/bonitasoft/engine/bpm/bar/ActorMappingContribution.java

```diff
@@ -35,4 +44,32 @@ public String getFileName() {
         return ACTOR_MAPPING_FILE;
     }
 
+    @Override
+    public boolean readFromBarFolder(final BusinessArchive businessArchive, final File barFolder) throws IOException {
+        final File file = new File(barFolder, ACTOR_MAPPING_FILE);
+        if (file.exists()) {
+            final byte[] content = IOUtil.getContent(file);
+            try {
+                businessArchive.setActorMapping(new ActorMappingConverter().deserializeFromXML(content));
+            } catch (JAXBException | org.xml.sax.SAXException e) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    @Override
+    public void saveToBarFolder(final BusinessArchive businessArchive, final File barFolder) throws IOException {
+        final ActorMapping actorMapping = businessArchive.getActorMapping();
+        if (actorMapping != null) {
+            try {
+                final byte[] fileContent = new ActorMappingConverter().serializeToXML(actorMapping);
```

Comment:
IMHO ActorMappingConverter should be passed to ActorMappingContribution constructor